configure

```diff
@@ -3413,10 +3414,13 @@ if test -n "$_host"; then
 			_zlib=yes
 			;;
 		*mingw32*)
-			_sdlconfig=$_host-sdl-config
+			_sdlconfig=$_host-sdl2-config
+			_libcurlconfig=$_host-curl-config
 			_windres=$_host-windres
 			_ar="$_host-ar cr"
 			_ranlib=$_host-ranlib
+			_strip=$_host-strip
+			_ldd="$_host-peldd -t --ignore-errors"
```

Comment:
This won't work if peldd is not present, and it is not on MXE.

Reply:
ACK, this will break MXE.

Reply:
I'm using MXE, and I do have `peldd`. Do you have `pe-util` installed?

Reply:
I don't and it's not part of the base installation. This means you need to check if binary is present.
Something like that could be used:
`command -v "$_host-peldd" >/dev/null 2>&1 && _ldd="$_host-peldd -t --ignore-errors"` could be used.
In the same way, ldd should not be set if it isn't available.

Reply:
Already done :)

And it won't break MXE anyway, since ldd doesn't work either. It is only used when making the `win32dist-mingw` target, which I'm probably the only one using it :)